examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -2361,6 +2361,29 @@ server cluster ModeSelect = 80 {
   command ChangeToMode(ChangeToModeRequest): DefaultSuccess = 0;
 }
 
+/** Attributes and commands for configuring the Refrigerator alarm. */
+server cluster RefrigeratorAlarm = 87 {
+  bitmap AlarmMap : BITMAP32 {
+    kDoorOpen = 0x1;
+  }
+
+  info event Notify = 0 {
+    AlarmMap active = 0;
+    AlarmMap inactive = 1;
+    AlarmMap state = 2;
+    AlarmMap mask = 3;
+  }
+
+  readonly attribute AlarmMap mask = 0;
+  readonly attribute AlarmMap state = 2;
```

Comment:
There is a missing Latch attribute, which means the RESET feature is not implementable. Is it indended that the RESET feature cannot be implemented with this version of server cluster?

Reply:
Spec says that RESET is always false in refrigerator alarm cluster.